src/View/Form/ContextInterface.php

```diff
@@ -72,6 +72,14 @@ public function val($field);
      */
     public function isRequired($field);
 
+    /**
+     * Get maximum length of a field from model validation
+     *
+     * @param string $field A dot separated path to check required-ness for.
+     * @return int|null
+     */
+    public function getMaxLength($field);
```

Comment:
We can't change the interface in minor release. Add a `@method` annotation instead at the top similar to what's done for `getRequiredMessage()`.

Reply:
OK i have changed